css/components/button.css

```diff
@@ -0,0 +1,116 @@
+/* button css  */
+.btn{
+    margin: 0.2rem;
+    min-width: 6.25rem;
+    width: max-content;
+    padding: .8rem 1rem ;
+    border-radius: 1px;
+    cursor: pointer;
+    font-weight: 400;
+    box-shadow: 2px 2px var(--light-black);
+}
+
+.btn-primary{
+    background-color: var(--primary-color);
+    color: var(--mint-white);
+}
+
+.btn-primary:hover{
+    background-color: var(--light-more-blue);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+.btn-success{
+    background-color: var(--success-color);
+    color: var(--mint-white);
+}
+
+.btn-success:hover{
+    background-color: hsl(144,100%,50%);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+.btn-warning{
+    background-color: var(--warning-color);
+    color: var(--mint-white);
+}
+
+.btn-warning:hover{
+    background-color: hsl(40,100%,65%);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+.btn-info{    
+    background-color: var(--info-color);
+    color: var(--mint-white);
+}
+
+.btn-info:hover{
+    background-color: hsl(195,83%,68%);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+
+.btn-error{    
+    background-color: var(--alert-color);
+    color: var(--dark-black);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+.btn-error:hover{
+    background-color: hsl(0,85%,75%);
+    color:var(--mint-white);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+.btn-border{
+    background-color: var(--mint-white);
+    color: var(--light-black);
+    border:1px solid var(--light-black);
+}
+
+.btn-border:hover{
+    border:1px solid var(--dark-black);
+    color:var(--dark-black);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+/* Link button */
+
+.btn-link{
+    background-color: var(--dark-blue);
+    color: var(--mint-white);
+}
+
+.btn-link a{
+    color: var(--dark-white);
+}
+
+
+.btn-icon{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    gap: 8px;
+}
+
+.btn-icon-item{
+    height: max-content;
+} 
+
+/* floating button */
+
+
+.btn-float{
+    margin: 0.2rem;
+    min-width: 2rem;
+    min-height: 2rem;
+    padding: 0.8rem;
+    border-radius: 50%;
+    text-align: center;
+    flex-direction: row;
+}
```

Comment:
As per the video, I think icons are not in the center. U can try it with flex to center it.

Reply:
i changed accoding you